pkg/services/repo_service.go

```diff
@@ -19,20 +23,22 @@ type RepositoryDB interface {
 }
 
 type argoCDService struct {
-	repositoriesDB 		RepositoryDB
-	repoClientset 		apiclient.Clientset
+	repositoriesDB RepositoryDB
+	repoClientset  apiclient.Clientset
 }
 
 type Apps interface {
```

Comment:
I think we should rename this interface to `Repos` instead of `Apps`, as we are doing more than just fetch `App` data now. We also call the reference of this interfaces instance as `repos`.

Reply:
+1 for that